### PR TITLE
[Fix]: React Charting Heat Map Chart tooltip appears above the wrong cell

### PR DESCRIPTION
**Pull request by harsh9975**
https://github.com/microsoft/fluentui/assets/33349461/f92ea51f-0e90-42bc-b27c-961dbda99310

## Previous Behavior

When you hover over a cell in the heat map, a tooltip appears. It is supposed to appear above that cell. However, for certain cells, it appears above the wrong cell.

## New Behavior

There was a small issue whenever we are hovering over index [1,10] the tooltip was showing in [11,0] cell. reason earlier it was storing both the index as string `110` and `110` which created the conflict. however i have addded `x1y10` and `x11y0` to differentiate.

## Related Issue(s)

#29740 
#29741 

- Fixes #

### CODE REVIEW

**Comment by harsh9975** [comment]
#29740

**Comment by harsh9975** [comment]
#29741

**Comment by harsh9975** [comment]
@microsoft-github-policy-service agree

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/ff6d3f116ad039e81dc6343bd2cd5dee861c7282'>ff6d3f116ad039e81dc6343bd2cd5dee861c7282</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "[Bug]: React Charting Heat Map Chart tooltip appears above the wrong cell fixed",
```
[the remark this answers is not in the document]
Comment (harsh9975) [reply change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json:3]: Hi @AtishayMsft , I am getting this error.
Validation Error:

  Module @fluentui/jest-serializer-merge-styles in the snapshotSerializers option was not found.
         <rootDir> is: /Users/harish/Desktop/opensource/fluentui/packages/react-charting

I encountered this while running `yarn update-snapshots` in react-charting file.

[the remark this answers is not in the document]
Comment (AtishayMsft) [reply change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json:3]: you need to run yarn update-snapshots from fluentui root path.

[the remark this answers is not in the document]
Comment (harsh9975) [reply change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json:3]: this time I did `yarn update-snapshots` from fluentui root path
again I'm getting this same error. though in the below error it shows root dir as `packages/utilities`
 Validation Error:
Module @fluentui/jest-serializer-merge-styles in the snapshotSerializers option was not found.
<rootDir> is: /Users/harish/Desktop/opensource/fluentui/packages/utilities

I referred installation guide from here https://github.com/microsoft/fluentui/wiki/Setup#basic-setup
and I haven't changed any test files.

[the remark this answers is not in the document]
Comment (AtishayMsft) [reply change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json:3]: You will need to run `yarn build` atleast once at the root to get things setup.

[the remark this answers is not in the document]
Comment (harsh9975) [reply change/@fluentui-react-charting-cc85e8df-5d8a-4a55-b0fe-7e5806540c7c.json:3]: done👍

**Review by AtishayMsft — APPROVED** [review]

**Comment by AtishayMsft** [comment]
@harsh9975 This fix is merged. Thanks for your contribution. Do let me know if you are interested to contribute to fix more issues in the fluent react charting library. We welcome contributions from the community.

**Comment by harsh9975** [comment]
@AtishayMsft yes, I am interested in contributing to more fixes.

**Comment by AtishayMsft** [comment]
Great. Feel free to join the fluent charting community @ https://aka.ms/FluentCharting/discord . We can discuss about few pending issues there.

**Pull request merged**